docs/doc_sources/physics_glossary.rst

```diff
@@ -39,6 +39,9 @@ Physics Glossary
   intrinsic impurity
     An impurity which is assumed to be already in the plasma (without being injected by us).
 
+  invmu_0_dLedR
+    :math:`\frac{1}{\mu_0}\frac{\partial L_e}{\partial R}` (eq.21 in :cite:`Barr_2018`)
```

Comment:
I can't really think of a more descriptive name that isn't absurdly long. I will describe how this is used though.
`inverse_mu_0_partial_external_inductance_partial_major_radius`???

Reply:
How is it used? Or if it isn't used, do we need to record its value as a result? i.e. we want outputs to be things that people can interpret and use. If it's only used internally, then combine the algorithms so that this value is only used internally

Reply:
It is not intended to be used directly, but I suppose it could be. In fact, @hassec made the point that I should remove it from the initialization file which I have done. I suppose I could also just remove it from the glossary if that makes it simpler. I think combining this function with the vertical magnetic field function would be kind of cluttered in the `/formulas/inductances.py`. Is that ok?

Reply:
You can combine in the algorithm instead of in the formula